pkg/codegen/schema/schema.go

```diff
@@ -52,6 +52,8 @@ const (
 	jsonType    primitiveType = 8
 )
 
+const object = "object"
```

Comment:
IMO this sort of constant is not particularly helpful. Was `goconst` complaining?

Reply:
yeah I didn't like it either, I can just turn off `goconst` instead.